bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoLightHandler.java

```diff
@@ -164,6 +153,42 @@ public void dispose() {
         return this.wemoBridgeHandler;
     }
 
+    private void poll() {
+        synchronized (jobLock) {
+            if (pollingJob == null) {
+                return;
+            }
+            try {
+                logger.debug("Polling job");
+
+                if (host.isEmpty()) {
+                    if (service != null) {
```

Comment:
Redundant null check (there's a compiler warning for this).

Reply:
Yes, I know, but without this nullcheck, service can be null and
URL descriptorURL = service.getDescriptorURL(this);
will throw an NPE

Reply:
If service can be null, please declare it `@Nullable`.

Reply:
Done in all handler classes.